src/view/trip-events-edit-point-view.js

```diff
@@ -177,10 +185,55 @@ export default class NewTripEventsEditPointView extends AbstractStatefulView {
     this.#submitDeletePoint();
   };
 
+  #dateFromChangeHandler = ([userDate]) => {
+    this.updateElement({
+      point: {
+        ...this._state.point,
+        dateFrom: userDate,
+      }
+    });
+  };
+
+  #dateToChangeHandler = ([userDate]) => {
+    this.updateElement({
+      point: {
+        ...this._state.point,
+        dateTo: userDate,
+      }
+    });
+  };
+
+  #setDatepickerStart() {
+    this.#datepickerStart = flatpickr(
+      this.element.querySelector('[name ="event-start-time"]'),
+      {
+        dateFormat: 'd/m/y h:i',
+        enableTime: true,
+        'time_24hr': true,
+        defaultDate: this._state.point.dateFrom,
+        onChange: this.#dateFromChangeHandler
+      }
+    );
+  }
+
+  #setDatepickerEnd() {
+    this.#datepickerStart = flatpickr(
+      this.element.querySelector('[name ="event-end-time"]'),
+      {
+        dateFormat: 'd/m/y h:i',
+        enableTime: true,
+        'time_24hr': true,
+        defaultDate: this._state.point.daTo,
```

Comment:
```suggestion
        defaultDate: this._state.point.dateTo,
        minDate: this._state.point.dateFrom,
```

аналогично для первого максимальная дата

Reply:
> аналогично для первого максимальная дата
> максимальная 

@Anzorik2024